src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,71 @@
-export const Pagination = () => {};
+import React from 'react';
+
+interface Props {
+  total: number;
+  currentPage: number;
+  onPageChange: (pageNumber: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  currentPage,
+  onPageChange,
+}) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= total; i += 1) {
+    pageNumbers.push(i);
+  }
```

Comment:
this cycle will be executed every time when state or props changed
so make sense to move this code to the function